src/plugins/data/public/search/aggs/types.ts

```diff
@@ -75,4 +101,25 @@ export type AggExpressionFunctionArgs<
  */
 export interface AggParamsMapping {
   terms: AggParamsTerms;
+  [METRIC_TYPES.AVG]: AggParamsAvg;
```

Comment:
Should be also registered in `src/plugins/data/public/search/aggs/agg_types.ts`

Reply:
added to agg_types.ts